apps/CMakeLists.txt

```diff
@@ -1,25 +1,46 @@
-# Copyright (c) 2013-2016, EPFL/Blue Brain Project
+# Copyright (c) 2013-2017, EPFL/Blue Brain Project
 #                          Raphael Dumusc <raphael.dumusc@epfl.ch>
 
+include(CommonHelp)
+
 include_directories(${PROJECT_SOURCE_DIR}/tide/core)
 
 add_subdirectory(TideForker)
 add_subdirectory(TideMaster)
 add_subdirectory(TideWall)
 add_subdirectory(Launcher)
 add_subdirectory(Whiteboard)
+
+set(_tide_in ${CMAKE_CURRENT_SOURCE_DIR}/tide)
+set(_tide_out ${PROJECT_BINARY_DIR}/bin/tide)
+add_custom_command(
```

Comment:
why not configure_file(@only)? This way it should run only if _tide_in has changed